hr_timesheet_sheet/tests/common.py

```diff
@@ -0,0 +1,227 @@
+# Copyright 2018-2020 ForgeFlow, S.L.
+# Copyright 2018-2019 Brainbean Apps (https://brainbeanapps.com)
+# Copyright 2018-2019 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo.tests.common import TransactionCase
+
+
+class HrTimesheetSheetCommon(TransactionCase):
```

Comment:
Please don't change the file, as this is not friendly for blame operations. Keep it everything in the same file.

Reply:
Hi Pedro don't blame me ;- ), I use setup in my PR.
That's the purpose of this PR

Check https://github.com/OCA/timesheet/pull/704

Reply:
Maybe you want I change class to setup but let code in the same file ?

Reply:
Yeah, that's what I'm stating: I want to split the test classes, but in the same file, not putting the `common.py` file.

Reply:
Now fixed